buildingmotif/api/views/model.py

```diff
@@ -54,7 +54,9 @@ def get_model_graph(models_id: int) -> Graph:
     except NoResultFound:
         return {"message": f"No model with id {models_id}"}, status.HTTP_404_NOT_FOUND
 
-    return model.graph.serialize(format="ttl"), status.HTTP_200_OK
+    g = Graph() + model.graph
```

Comment:
There's also a `copy_graph` function in `utils.py` however, looking at the implementation it seems like it might be slower than this. @gtfierro what do you think?